packages/table/src/Cell/Cell.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { PropsWithChildren } from 'react';
+import PropTypes from 'prop-types';
+
+import { CellProps } from './Cell.types';
+import InternalCellBase from './InternalCellBase';
+import InternalCellWithRT from './InternalCellWithRT';
+
+const Cell = <T extends unknown>({
+  cell,
+  ...rest
+}: PropsWithChildren<CellProps<T>>) => {
+  const usesRT = !!cell;
```

Comment:
should this value be coming from context? single source of truth

Reply:
The user is the one calling the hook and we can't know which `cell` object in the `data` prop the `Cell` component needs, so the user does need to pass it in directly to the component